ivy/functional/ivy/elementwise.py

```diff
@@ -4038,6 +4038,89 @@ def positive(
     ret
         A new array with the positive value of each element in ``x``.
 
+         Functional Examples
+    -------------------
+
+    With: code:`ivy.Array` input:
+
+    >>> x = ivy.array([2,3,5,7])
+    >>> y = ivy.positive(x)
+    >>> print(y)
+    ivy.array([2, 3, 5, 7])
+
+    >>> x = ivy.array([0,-1,-0.5,2,3])
+    >>> y = ivy.zeros(5)
+    >>> ivy.positive(x,out=y)
+    >>> print(y)
+    ivy.array([0., -1., -0.5,  2.,  3.])
+
+    >>> x = ivy.array([[1.1,2.2,3.3], \
```

Comment:
To clarify, i mean keep the spacing consistent in all the array lists, e.g. `>>> x = ivy.array([[1.1,2.2,3.3], \` should be `>>> x = ivy.array([[1.1, 2.2, 3.3], \`